sentinel/tests/unit/schedule/index.js

```diff
@@ -324,4 +329,66 @@ describe('scheduler', () => {
     });
   });
 
+  // Regression test to prevent again jobs launched 1 sec before time
+  // to NOT being skipped (https://github.com/medic/cht-core/issues/6634)
+  describe('Scheduling without gaps', () => {
```

Comment:
I thought the same at the beginning, but actually this is very related with the `init()` method called in the `index.js` method, because depending of the time when `init()` is called the error could happen or not with the previous code. Still if you think is more appropriate to move it there I'll do so.